Meadowlark-js/services/meadowlark-fastify/.env.example

```diff
@@ -56,4 +56,9 @@ TOKEN_URL: http://localhost:3000/local/api/oauth/token
 # Valid levels include: ERROR, WARNING, INFO, DEBUG
 LOG_LEVEL=DEBUG
 # When true, logs to plain text instead of JSON
-IS_LOCAL=true
\ No newline at end of file
+IS_LOCAL=true
+
+
+#### Authorization Server options
+# Configurable authorization store plugin - set to an npm package name
+AUTHORIZATION_STORE_PLUGIN=@edfi/meadowlark-mongodb-backend
```

Comment:
Will need to add to existing .env files.

Reply:
@andonyns @stevenarnold-ed-fi please note ^^^^